src/i_video.c

```diff
@@ -1794,7 +1790,8 @@ void I_InitGraphics(void)
     {
         I_Error("Failed to initialize video: %s", SDL_GetError());
     }
-    SDL_SetHint("SDL_HINT_RENDER_BATCHING", "0");
```

Comment:
It should be `SDL_HINT_RENDER_BATCHING` without the quotes. So that line didn't actually do anything (I checked with `SDL_GetHint`).

Reply:
Good catch. We were originally testing it correctly here: https://github.com/fabiangreffrath/woof/issues/1685#issuecomment-2134358231

Reply:
So, do we need `SDL_HINT_RENDER_BATCHING`? It seems there is no difference for me.

Reply:
I think we should leave it out. Power users can mess with that setting if they want, and it's gone in SDL3 anyway.